src/main/java/org/xbill/DNS/DNSSession.java

```diff
@@ -0,0 +1,129 @@
+package org.xbill.DNS;
```

Comment:
Please add `// SPDX-License-Identifier: BSD-2-Clause` to new files to clearly indicate their license.

Reply:
Sure. How do we feel about adding copyright statements? 

I guess the main purpose for such things would be if someone would make an effort to contact all copyright holders to investigate the viability of a license change in the future. I am happy with or without, and I'm happy with 2 clause BSD.

Reply:
The Git history is enough to follow the copyright. The Kernel and systemd do it the same with the simple SPDX comment.